src/allocator_thread.c

```diff
@@ -86,14 +86,14 @@ static ip_type4 ip_from_internal_list(char* name, size_t len) {
 	}
 	// grow list if needed.
 	if(internal_ips->capa < internal_ips->counter + 1) {
-		PDEBUG("realloc\n");
+		PDEBUG1("realloc\n");
 		new_mem = realloc(internal_ips->list, (internal_ips->capa + 16) * sizeof(void *));
 		if(new_mem) {
 			internal_ips->capa += 16;
 			internal_ips->list = new_mem;
 		} else {
 	oom:
-			PDEBUG("out of mem\n");
+			PDEBUG1("out of mem\n");
```

Comment:
you're polishing a turd. what i want to say is that this is dead code, that only gets activated if a user explicitly compiles the code with -DDEBUG (there's not even a configure switch for that).
apart from that the change you propose is utterly verbose since you not just duplicate the variadic code, you even change the toplevel macro name so all calling sites need to be touched!